de.bund.ausweisapp.ausweisapp2.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: de.bund.ausweisapp.ausweisapp2
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: AusweisApp2
+finish-args:
```

Comment:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: iris
```

Sounds like it also needs `--device=dri` for graphical acceleration. Other than this, LGTM.

Reply:
Oh so that's why it suddenly stopped working for me! Is this a recent change in flatpak, explicitly having to require `dri`? Anyways, I took out the environment variable hardcoding software rendering along the added capability as GL rendering does work, again, this way.

Reply:
I think it's been always like this, maybe some recent runtime change triggered it?